tests/study/test_repository.py

```diff
@@ -1004,3 +1177,71 @@ def test_update_tags(
     # Check that only "Tag1" and "Tag3" are present in the database
     tags = db_session.query(Tag).all()
     assert {tag.label for tag in tags} == {"Tag1", "Tag3"}
+
+
+@pytest.mark.parametrize(
+    "managed, study_ids, exists, expected_ids",
+    [
+        (None, [], False, {"5", "6"}),
```

Comment:
But it seems that this duplicates the first test of the file :/ Or i'm missing something

Reply:
@MartinBelthle  
I could not put this test in the listing class as the purpose is not the same. 
Counting returns an `int` meanwhile listing returns a list of `Studies`.
Regarding the similarity with previous part is partially due to the fact that @laurent-laporte-pro  asked only for general case.